src/__tests__/index.test.ts

```diff
@@ -1,5 +1,420 @@
-describe('example test', () => {
-  it('works', () => {
-    expect(1 + 1).toEqual(2)
+import { ApiClient } from '../index'
```

Comment:
This test file is getting really big. What I like in such cases is to split them based on `describe` blocks to make it easier to follow. WDYT?

Reply:
already added `describes` for the 4 main methods. I'll group it more 👍